packages/edit-site/src/components/header/tools-more-menu-group/index.js

```diff
@@ -10,7 +10,7 @@ import { createSlotFill, MenuGroup } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 const { Fill: ToolsMoreMenuGroup, Slot } = createSlotFill(
-	'ToolsMoreMenuGroup'
+	'EditSiteToolsMoreMenuGroup'
```

Comment:
I personally think that those names should become private for the SlotFill and we only keep old names for backward comaptibility.